src/macaron/repo_finder/repo_finder_deps_dev.py

```diff
@@ -86,25 +84,17 @@ def create_urls(self, group: str, artifact: str, version: str) -> list[str]:
         list[str]
             The list of created URLs.
         """
-        package_name = f"{group}:{artifact}@{version}"
```

Comment:
Why is this variable removed here? Is it not used anywhere and was left here by accident?

Reply:
It was only used for debug output relating to a failed GET request. That is now handled as part of the send_get_http_raw function call, so there was no need to keep it here.